tests/input/obo-compliance.obo

```diff
@@ -658,6 +705,20 @@ subset: S {source="PMID:123464"}
 id: X:1
 disjoint_from: X:2 {source="PMID:123465"}
 
+!! name: gci-relation
+!! description: General Class Inclusion relation
+
+[Term]
+id: X:1
+relationship: R:1 X:2 {gci_predicate="R:2", gci_filler="X:3"}
+
+!! name: gci-is-a
+!! description: General Class Inclusion is_a
+
+[Term]
+id: X:1
+is_a: X:2 {gci_predicate="R:2", gci_filler="X:3"}
```

Comment:
Shouldn’t that be `gci_relation` instead of `gci_predicate`? I don’t think I have ever seen that `gci_predicate` qualifier, and the OBO spec does not mention it at all.